package.json

```diff
@@ -88,6 +89,7 @@
     "eslint": "7.3.0",
     "eslint-config-prettier": "6.11.0",
     "eslint-plugin-prettier": "3.1.4",
+    "expect": "^26.1.0",
```

Comment:
Everything we should need is shipped with Jest already. This isn't ideal to add another dependency here, but maybe this is the easiest way to start. I've also tried to use `expect` and `test` from jest internal environment and it was not easy

Reply:
I think they don't expose the classes in their .d.ts files, that's why the standard `jest` package was not enough.

Reply:
Not sure, definitely more research is needed on more native Jest integration